main.cpp

```diff
@@ -162,7 +162,7 @@ rickAstley.runAroundAndDesertYou();
 //  10)
 //  Noun: Cyberaktiv
 //  action 1: Cyberaktiv learns c++
-cyberaktiv.learnCpp();
+cyberaktiv.learnsCpp();

```

Comment:
Pick a different action for this noun.
Change the plain text version and the pseudo-code
- [ ] revise